src/main/resources/io/jenkins/plugins/analysis/core/steps/IssuesRecorder/config.properties

```diff
@@ -5,4 +5,7 @@ title.enabledForFailure=Run always
 description.enabledForFailure=Enable recording for failed builds.
 
 title.aggregatingResults=Aggregate results
-description.aggregatingResults=Aggregate results of all tools into a single result.
\ No newline at end of file
+description.aggregatingResults=Aggregate results of all tools into a single result.
```

Comment:
The same here, into `publish-parameters.properties`

Reply:
Resolved :+1: